frontend/cypress/e2e/search.cy.js

```diff
@@ -42,6 +42,6 @@ describe("Search feature", () => {
     cy.get("div")
       .contains(/teaching assistant/i)
       .should("be.visible")
-    SearchPage.resultsLabel().contains(3)
+    SearchPage.resultsLabel().contains(/teacher and web developer/i)
```

Comment:
```suggestion
    SearchPage.resultsLabel().contains(/teacher and web developer/i).contains(3)
```
check for both the number and the text otherwise we risk regressing on the first one